anago

```diff
@@ -864,13 +908,15 @@ if ((FLAGS_nomock)); then
   # Override any --noclean setting if nomock
   FLAGS_noclean=0
   GCRIO_REPO="${FLAGS_gcrio_repo:-google_containers}"
+  ALL_CONTAINER_REGISTRIES=("$GCRIO_REPO" "kubernetes-release-test")
 else
   # This is passed to logrun() where appropriate when we want to mock
   # specific activities like pushes
   LOGRUN_MOCK="-m"
   # Point to a $USER playground
   RELEASE_BUCKET+=-$USER
   GCRIO_REPO="${FLAGS_gcrio_repo:-kubernetes-release-test}"
+  ALL_CONTAINER_REGISTRIES=("$GCRIO_REPO" "google_containers")
```

Comment:
I think this needs to be `google-containers`, not `google_containers`, because the "real" name matches the project, `google-containers`. `gs://artifacts.google_containers.appspot.com` does not exist.